web/instances/instances.go

```diff
@@ -427,4 +455,6 @@ func Routes(router *echo.Group) {
 	router.POST("/:domain/import", importer)
 	router.POST("/:domain/orphan_accounts", cleanOrphanAccounts)
 	router.POST("/redis", rebuildRedis)
+	router.GET("/prefix/:domain", showPrefix)
+	router.GET("/swift-prefix/:domain", getSwiftBucketName)
```

Comment:
I'd prefer to have `/:domain/prefix` and `/:domain/swift-prefix`, to be consistent with the other routes in this file